examples/webgpu_cubemap_adjustments.html

```diff
@@ -72,6 +72,12 @@
 				cube2Texture.generateMipmaps = true;
 				cube2Texture.minFilter = THREE.LinearMipmapLinearFilter;
 
+				renderer = new THREE.WebGPURenderer( { antialias: true } );
+				await renderer.init();
+
+				const cube2Blur = new THREE.CubeRenderTarget(cube2Texture.source.data[0].width);
+				cube2Blur.fromCubeTexture(renderer, cube2Texture, 0.1);
```

Comment:
I'm throwing in a fixed-size blur for now to test - to have it hooked up to the `backgroundBlur` UI I believe we'll need some kind of `cubeBlurNode` that works a bit like `pmremNode`, automatically regenerating the texture as needed.